Code/CheXpertDataset.py

```diff
@@ -43,28 +48,41 @@ def __len__(self):
 
     def __getitem__(self, idx):
         image_path = self.df.iloc[idx, 0]
-        im = Image.open(os.path.join(self.root_dir, image_path))
-        label = self.df.iloc[idx, 1:]
+        # im = Image.open(os.path.join(self.root_dir, image_path))
+        im = load_img(os.path.join(self.root_dir, image_path),target_size=(600,600), interpolation='nearest')
 
+        label = self.df.iloc[idx, 1:]
         if self.label_transform is not None:
-            label = self.label_transform(label)
+            # label = self.label_transform(label.values)
+            self.label_transform(label)
 
         if self.image_transform is not None:
             im = self.image_transform(im)
 
-        return im, label
+        return im, torch.from_numpy(np.array(label[label_columns].values.tolist()))
 
 
 class ReplaceNaNTransform:
     def __call__(self, sample):
-        return sample[label_columns].fillna(-2)
-
+        sample[label_columns].fillna(-2)

```

Comment:
I don't see how this change would work.  According to the documentation, `fillna` is by default non-desctructive

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.fillna.html

Reply:
looks like the only thing changed was the return being removed so the behaviour hasn't changed right? 
fillna being the right answer here could be a good question to ask, but i don't think it's worth holding up the pr.

Reply:
oh.. i see the return below.

Reply:
based on the docs we need to do this: 
```suggestion
        sample[label_columns].fillna(-2, inplace=True)
```